### PR TITLE
[prop] Add function to read old Values (incl. uncertainty, reference)

### PR DESCRIPTION
**Pull request by hkchekc**
Refer to Issue #364 
Originally, when reading values (getting props.values) from file with version < 1.1.1, it will return an old style compound type with (v.value, v.uncertainty, v.reference, v.filename, v.encoder, v.checksum). 

I break it down for value, uncertainty and reference. All now returns a tuple for values, single value for reference and uncertainty.

The problem is we cannot distinguish between an empty str or 0.0 and an unsetted uncertainty or reference.

### CODE REVIEW

File: nixio/property.py [hunk 1]
```diff
@@ -145,6 +145,12 @@ def unit(self, new):
 
     @property
     def uncertainty(self):
+        dataset = self._h5dataset
+        x, y, z = dataset._parent.file.attrs["version"]
+        if y < 1 or z < 1:
```
Comment (achilleas-k) [inline nixio/property.py]: Best to check if the entire ver tuple is `< (1, 1, 1)`.

File: nixio/property.py [hunk 2]
```diff
@@ -215,9 +227,21 @@ def odml_type(self, new_type):
 
         self._h5dataset.set_attr("odml_type", str(new_type))
 
+    def _read_old_values(self):
+        val = self._h5dataset.dataset[:]
+        val_tu = tuple()
+        for v in val:
+            v = v["value"]
+            val_tu += (v,)
+        return val_tu
```
Comment (achilleas-k) [inline nixio/property.py]: @jgrewe @gicmo What's the current situation in NIX with reading old-style values and is there any plan to change it?  Do we ignore the extra information and just return a _new style_ tuple of raw values?

File: nixio/property.py (hunk 1 above)
Comment (hkchekc) [reply nixio/property.py]: resolved

File: nixio/property.py (hunk 2 above)
Comment (jgrewe) [reply nixio/property.py]: at the moment we just do that, returning a vector of Variant, i.e. the raw values, reference was swept under the rug, methinks...
need to remedy that

Comment (achilleas-k) [reply nixio/property.py]: We can go ahead and merge then and see how it develops in NIX.

**Pull request merged**